src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts

```diff
@@ -1,67 +1,44 @@
-import {async, ComponentFixture, getTestBed, TestBed} from '@angular/core/testing';
```

Comment:
can we rename the file withe same for consistency please?
We have several tech-record-*
Here it will make it hard to search

Reply:
As agreed wit the team, code was delivered prior to ADR and reference ticket for future refactor if needed

Reference: https://jira.dvsacloud.uk/browse/CVSB-15101